### DISCUSSION TITLE
Несколько vpn соединений

### DISCUSSION THREAD

**Discussion by Andrey999r** [qzeleza/kvas#203] [Ideas]
Я бы добавил возможность использования кваса для нескольких vpn соединений.1 маловато

**Comment by Ajivitca** [comment]
и как бы полезно для xxxxx чтоб трафик не шел только через один сервер

**Comment by Igor-178** [comment]
Поддержка двух (нескольких) VPN соединений, с возможностью балансировки и / или failover.
